express.js

```diff
@@ -517,15 +517,14 @@ module.exports = (app) => {
 			'items' //some widgets double-nest 'items'
 		];
 
-		const nonstandard_props = [];
+		let nonstandard_props = [];
```

Comment:
why `const` -> `let` ?

`const` is used elsewhere here, so it's not a node support thing, the array contents change, but `nonstandard_props` isn't re-assigned, so it should be fine, no?

Not a blocking thing, but looks like it _should_ be `const`

Reply:
Not positive what I was doing here. I think I might have been planning on using the same array to store nonstandard props on all of the `items` layers, which would have meant resetting it to an empty array after each pass - but I didn't end up doing that and probably just forgot to go back to using `const`.